apps/explorer/src/ui/TransactionBlockCard.tsx

```diff
@@ -66,7 +66,7 @@ function TransactionBlockCardHeader({
 
     if (collapsible) {
         return (
-            <Disclosure.Button as="div" className="w-full">
+            <Disclosure.Button as="div" className="w-full cursor-pointer">
```

Comment:
having this rendered as a `button` has some strange CSS behavior. Need to turn into a div with a hover cursor